### ISSUE TITLE
Problem installing skylighting on Windows

### ISSUE THREAD

**Issue by dbobak** [jgm/skylighting#4]
When trying to install skylighting on Windows 10, 64bit, I'm getting the following error:

```
c:\users\deni>cabal install skylighting
Resolving dependencies...
Configuring skylighting-0.1.1.2...
Building skylighting-0.1.1.2...
Failed to install skylighting-0.1.1.2
Build log ( C:\Users\deni\AppData\Roaming\cabal\logs\skylighting-0.1.1.2.log ):
Building skylighting-0.1.1.2...
Preprocessing library skylighting-0.1.1.2...
[  1 of 132] Compiling Skylighting.Regex ( src\Skylighting\Regex.hs, dist\build\Skylighting\Regex.o )
[  2 of 132] Compiling Skylighting.Types ( src\Skylighting\Types.hs, dist\build\Skylighting\Types.o )
[  3 of 132] Compiling Skylighting.Tokenizer ( src\Skylighting\Tokenizer.hs, dist\build\Skylighting\Tokenizer.o )
[  4 of 132] Compiling Skylighting.Parser ( src\Skylighting\Parser.hs, dist\build\Skylighting\Parser.o )
[  5 of 132] Compiling Skylighting.Styles ( src\Skylighting\Styles.hs, dist\build\Skylighting\Styles.o )
[  6 of 132] Compiling Skylighting.Format.HTML ( src\Skylighting\Format\HTML.hs, dist\build\Skylighting\Format\HTML.o )
[  7 of 132] Compiling Skylighting.Format.LaTeX ( src\Skylighting\Format\LaTeX.hs, dist\build\Skylighting\Format\LaTeX.o )

src\Skylighting\Format\LaTeX.hs:124:24:
    No instance for (PrintfArg Text) arising from a use of `printf'
    In the second argument of `($)', namely
      `printf "\\colorbox[rgb]{%0.2f,%0.2f,%0.2f}{%s}" r g b x'
    In the expression:
      Text.pack $ printf "\\colorbox[rgb]{%0.2f,%0.2f,%0.2f}{%s}" r g b x
    In a case alternative:
        Just (r, g, b)
          -> Text.pack
             $ printf "\\colorbox[rgb]{%0.2f,%0.2f,%0.2f}{%s}" r g b x

src\Skylighting\Format\LaTeX.hs:130:25:
    No instance for (PrintfArg Text) arising from a use of `printf'
    In the second argument of `($)', namely
      `printf "\\textcolor[rgb]{%0.2f,%0.2f,%0.2f}{%s}" r g b x'
    In the expression:
      Text.pack
      $ printf "\\textcolor[rgb]{%0.2f,%0.2f,%0.2f}{%s}" r g b x
    In a case alternative:
        Just (r, g, b)
          -> Text.pack
             $ printf "\\textcolor[rgb]{%0.2f,%0.2f,%0.2f}{%s}" r g b x
cabal: Error: some packages failed to install:
skylighting-0.1.1.2 failed during the building phase. The exception was:
ExitFailure 1
```

**Comment by jgm** [comment]
What version of ghc are you using?

+++ Darek Bobak [Jan 30 17 03:04 ]:

**Comment by dbobak** [comment]
Updating ghc did the trick. Thank you!

**Comment by jgm** [comment]
I'm going to fix it for older ghc versions anyway.


+++ Darek Bobak [Jan 30 17 06:10 ]:

**Comment by jgm** [comment]
Should be fixed with 0.1.1.3.
